packages/agent-utils/src/embeddings/LocalVectorDB.ts

```diff
@@ -0,0 +1,50 @@
+import { EmbeddingApi } from "./EmbeddingApi";
+import { normalize, normalizedCosineSimilarity } from "./utils";
+import { LocalDocumentStore } from "./LocalDocumentStore";
+import { LocalDocument } from "./LocalDocument";
+
+export class LocalVectorDB {
+  constructor(
+    private embeddingApi: EmbeddingApi,
+    private store: LocalDocumentStore,
+  ) {}
+
+  async add(items: {
+    text: string
```

Comment:
Could we do `add(items: string[])`?
If not, can we at least get an overload like that?